doc/Overview/Reactive/How-tos/SimpleFeed.md

```diff
@@ -0,0 +1,104 @@
+---
+uid: Overview.Reactive.HowTos.SimpleFeed
+---
+
+# How to create a simple feed
+
+In this tutorial you will learn how to create an MVUX project that displays asynchronous data (weather info) from a service,
+on-demand (via a refresh button).
+
+1. Create an MVUX project by following the steps in [this](xref:Overview.Reactive.HowTos.CreateMvuxProject) tutorial,
+and name your project `WeatherApp`.
+
+1. Add a class named *DataStore.cs*, and replace its content with the following:
+
+    ```c#
+    namespace WeatherApp;
+
+    public partial record WeatherInfo(int Temperature);
+
+    public class WeatherService
+    {       
+        public async ValueTask<WeatherInfo> GetCurrentWeatherAsync(CancellationToken ct)
+        {
+            // Fake delay to immitate requesting from a remote server
+            await Task.Delay(TimeSpan.FromSeconds(2), ct);
+
+            // assign a random number ranged -40-40.
+            var temperature = new Random().Next(-40, 40);
+
+            return new WeatherInfo(temperature);
+        }
+    }
+    ```
+
+    We're using [records](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record) in purpose,
+    as records are immutable and ensure purity of objects. Records also implements easy equality comparison and hashing.
+
+1. Create a file named *WeatherModel.cs* replacing its content with the following:
+
+    ```c#
+    public partial record WeatherModel
+    {
+        private readonly WeatherService _weatherService = new();
+    
+        public IFeed<WeatherInfo> CurrentWeather => Feed.Async(_weatherService.GetCurrentWeatherAsync);
+    }
+    ```
+
+    MVUX's analyzers will read the `WeatherModel` and will generate a special mirrored `BindableWeatherModel`,
+    which provides binding capabilities for the View, so that we can stick to sending update message in an MVU fashion.
+    
+    The `CurrentWeather` property value also gets cached, so no need to worry about its being created upon each `get`.
```

Comment:
Check accuracy of this - IFeed doesn't maintain any state, so any caching would be done in the FeedView from what I understand